video/out/vo_gpu.c

```diff
@@ -224,6 +224,9 @@ static int control(struct vo *vo, uint32_t request, void *data)
         p->ctx->fns->reconfig(p->ctx);
         resize(vo);
         return true;
+    case VOCTRL_GET_WANTS_DR:
+        *(bool*) data = !(p->ctx->ra->caps & RA_CAP_SLOW_DR);
+        return true;
```

Comment:
`gpu_next` will need this too, but not sure what it should look like.

the only other vo with this is `libmpv` which I guess should just duplicate this.

Reply:
https://code.videolan.org/videolan/libplacebo/-/merge_requests/336

Reply:
@CounterPillow still missing for libmpv